Improve description of loc in metadata (#272)
apache/accumulo-website: _docs-2/troubleshooting/system-metadata-tables.md

    key/value pair, and is 186 bytes long.

* `last:13fe86cd27101e5 []    127.0.0.1:9997` -

>     The last location data was written locally. Data was last written on 127.0.0.1:9997, and the

Reviewer: ```suggestion
    The last location data was written locally (from a minor or major compaction). Data was last written on 127.0.0.1:9997, and the
```

    unique tablet server lock data was `13fe86cd27101e5`. The default balancer

>     will tend to put tablets back on their last location where files writes have occurred to

Reviewer: ```suggestion
    will tend to assign tablets to the last location where its files have been written to
```

    attempt to improve data locality.
 
* `loc:13fe86cd27101e5 []    127.0.0.1:9997` -

>     Last assigned location for this tablet.  It was last held on 127.0.0.1:9997, and the

Reviewer: ```suggestion
    Last assigned location for this tablet (it's current location, if it is still hosted). It was last assigned to 127.0.0.1:9997, and the
```

>     attempt to improve data locality.

Reviewer: I think these descriptions are backwards. The "last" field is the most recent location written locally, and the "loc" field is the current location (most recently assigned).

I'm not certain about the behavior of the default balancer, but I think at least *some* balancers will use the "last" field in order to try to achieve data locality.
